templates/Dockerfile-template.ubuntu1804

```diff
@@ -35,10 +35,11 @@ RUN      buildDeps="autoconf \
                     pkg-config \
                     python \
                     libssl-dev \
+                    tclsh \
                     yasm \
                     zlib1g-dev" && \
         apt-get -yqq update && \
-        apt-get install -yq --no-install-recommends ${buildDeps}
+        DEBIAN_FRONTEND=noninteractive apt-get install -yq --no-install-recommends ${buildDeps}
```

Comment:
I would not add that : 
https://docs.docker.com/engine/faq/#why-is-debian_frontendnoninteractive-discouraged-in-dockerfiles

Reply:
Good to know that. BTW, I couldn't move forward without it when I tried to build an image for ubuntu18.04. It's asking for an input but I couldn't enter any input. How did you solve this problem?

Reply:
libsrt requires tclsh, and tclsh requires tzdata which causes the build hang.
I read your linked page, and I think my approach is safe as it only affects that 'apt-get install' line but not effective across the images (it is not configured as ENV).

Reply:
I see, but not put it in templates/Dockerfile-template.ubuntu1604 and other ubuntu variants, the build passes on all variants, I don't think that part is necessary, it's ok to have a warning.

Reply:
at least, ubuntu18.04 requires it. I tried to build for ubuntu18.04 and it stuck at installing tzdata package.

without `DEBIAN_FRONTEND=noninteractive`:
```
...
Setting up libnghttp2-14:amd64 (1.30.0-1ubuntu1) ...
Setting up mime-support (3.60ubuntu1) ...
Setting up libsigsegv2:amd64 (2.12-1) ...
Setting up libldap-common (2.4.45+dfsg-1ubuntu1.5) ...
Setting up libuv1:amd64 (1.18.0-3) ...
Setting up libreadline7:amd64 (7.0-3) ...
Setting up libpsl5:amd64 (0.19.1-5build1) ...
Setting up tzdata (2019c-0ubuntu0.18.04) ...
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
Configuring tzdata
------------------

Please select the geographic area in which you live. Subsequent configuration
questions will narrow this down by presenting a list of cities, representing
the time zones in which they are located.

  1. Africa      4. Australia  7. Atlantic  10. Pacific  13. Etc
  2. America     5. Arctic     8. Europe    11. SystemV
  3. Antarctica  6. Asia       9. Indian    12. US
Geographic area: 

^C
```

Reply:
ok we are probably making it a bigger deal than what it is, but @GnaphronG and I had a look and you can avoid `tclsh` installation if you apply that patch :

```
-        ./configure --prefix="${PREFIX}" && \
+        cmake -DCMAKE_INSTALL_PREFIX="${PREFIX}" . && \
```

give it a try.

With no tclsh, 18.04 doesn't try to install tzadata.

Reply:
I tried the patch and it worked. great!

BTW, if this change is applied to `Dockerfile-run`, then it'll affect all Dockerfiles generated from `update.py`. So instead of running `configure` script, which requires tclsh, we directly run `cmake` to build `libsrt`. Is it safe to skip the whole `configure` steps now and in the future?

If you think it's fine, I'll update `Dockerfile-run` like your proposal and remove `tcl`/`tclsh` from `buildDeps` in all `Dockerfile-template` files. (No need to run `configure` means no need to install `tcl`/`tclsh`.)

I still personally like `DEBIAN_FRONTEND=noninteractive apt-get install tclsh` approach because It's just a variable only affects to `apt-get install tclsh` in ubuntu18.04. No built images have side effect and no other variations are affected.

Reply:
I hear you, but if you look at https://github.com/Haivision/srt/blob/master/configure it's not doing much but passing arguments back to cmake, so safe to skip (famous last words ;-)

Yeah no tcl/tclsh means no tzdata , another approach would have been to set `TZ=UTC`.

Let's see how it goes with that approach.

Reply:
not necessary now that we don't have tclsh

Reply:
forgot to remove `DEBIAN_FRONTEND=noninteractive`. now removed!